src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = (
+  { todo: { completed, title, user } },
+) => (
+  <article
+    className={cn('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    <UserInfo user={user} />
```

Comment:
`user` might be `null` - see `getUserById` function
you should handle this case to avoid potential errors